src/stores/businessStore.ts

```diff
@@ -57,6 +58,11 @@ export const useBusinessStore = defineStore('business', {
       return DateUtilities.apiToDate(state.businessInfo.foundingDate)
     },
 
+    /** The start date. */
+    getStartDate (state: BusinessStateIF): Date {
+      return DateUtilities.apiToDate(state.businessInfo.startDate)
```

Comment:
It might be simpler to NOT convert this value in multiple places. What do you think?

Reply:
Please see my recent commit. Would something like that work better?